### ISSUE TITLE
Wireless only AMT Platforms not working with CIRA

### ISSUE THREAD

**Issue by datapharmer** [Ylianst/MeshCentral#112]
On wireless only AMT 11 platforms the CIRA script may result in AMT that does not connect because it does not toggle Wireless Settings to Enabled in S0, Sx/AC. Local mesh commander would also no longer connect (since intranet domain is set to a bogus domain and presumably local access is blocked?) so the only option I could find was to reset AMT config completely from pre-boot, use Mesh Commander to Enable Wireless Connections and then run the  CIRA script again. 

Even with this, AMT can only be used when the operating system is running and wifi adapter is connected to a valid network (Fast Call for Help fails and AMT stops responding if a dock is connected/wifi turned off) as it appears out-of-band wifi is not functioning properly when profiles are manually added and profiles are not syncing with Windows 10 OS. I've filed case# 04077994 with Intel to investigate the out-of-band/profile syncing issues.

This problem can be reliably reproduced using a Dell Latitude 7390 2-in-1 with latest firmware and drivers (tested on 4 machines out of box, after Dell website updates, after Dell Command updates, and with drivers directly from Intel).

**Comment by Ylianst** [comment]
Hi. I am working this week on CCM activation of AMT using the MeshAgent, once I get this done, I will take a look at this at some point. What you are describing is exactly correct because the CIRA script does not activate wireless. Even with AMT wireless activated, if the OS is not associated with an AP, AMT will not have connectivity unless it has WIFI profiles of it's own.

If I remember correctly, WIFI profiles where only sync'ed with AMT when entered using Intel ProSet which does not exist anymore (I would be wrong on this). So, there is really no option but to enter the WIFI profiles into AMT using MeshCommander.

Lastly, if you get stuck again with CIRA enabled and no connectivity, you can use MeshCommander locally with LMS. Add "127.0.0.1" in MeshCommander and you should be able to clear the CIRA settings without having to reset AMT.

In summary, yes, got to work to do on MeshCentral to auto support this usage.

**Comment by datapharmer** [comment]
Ok, so the issue and solution are as follows:
AMT Configuration from the MEBx or USB key does not enable the AMT setting LocalProfileSynchronizationEnabled, and a supplemental configuration profile is needed.

This can be scripted using the ACUConfig tool and the Profile.xml from the ACUWizard folder.
ACUConfig.exe /verbose /ouptut console ConfigAMT Profile.xml /DecryptionPassword YOURPASSWORD

Is it possible to add this enablement into the CIRA setup script?

**Comment by Ylianst** [comment]
MeshCentral no longer uses CIRA setup scripts to get CIRA configured. Setup scripts are not secured compared to end-to-end TLS configuration performed by the MeshCentral server today and so, these scripts where removed. If you are managing a Intel AMT computer within an agent-less device group, you can configure it using the 'meshcmd amtconfig' command that will be given to you by MeshCentral.

For "LocalProfileSynchronizationEnabled", this is always enabled by MeshCentral, both when managed with or without and agent.

I will close this issue since it's old and CIRA scripts are not longer supported, but feel free to open a new issue with any requests or questions you may have. Thanks.

**Comment by AFjolle** [comment]
Hi @Ylianst,
"LocalProfileSynchronizationEnabled" is crashing a lot of our machines LMS Service, because of the missing ProSet functionality. Is there a way to disable this from the config.json, i cant seem to find a parameter for that?

Thanks in advance

**Comment by si458** [comment]
@AFjolle quickly looking at the code, sadly its hard coded in the `amtmanager.js`
https://github.com/Ylianst/MeshCentral/blob/a6acb35a315a92194fd7a5cf3e6ca27adf2eeec0/amtmanager.js#L1883-L1889
so you would have to edit the `amtmanager.js` file to change that behaviour and then restart your meshcentral server
OR
if its as simple as changing a value, you can create an enhancement request and i could at look at adding the value into the AMt section of config.json to say use it or not?

**Comment by AFjolle** [comment]
@si458 Thanks for the quick response.

I simply changed the value to zero and did a restart of the server, and now every client is online again.

A way to disable this functionality would be great. Please tell me where to post an enchancement request.
Additionally options to choose which power states AMT is enabled in, would be a great addition too. We used that a lot in Intel EMA.
![image](https://github.com/user-attachments/assets/6e90968d-58a1-4bb3-a1a5-54e325955651)

As i have understood from my contact with Intel, the parameter was used back when the ProSet client/driver was still used, and this allowed the WiFi-profiles to be synced from ProSet to the server.
Why it is now causing clients to crash i'm uncertain about, but i would guess it has something to do with which WiFi driver is installed on the system, or maybe a conflict with the Windows "settings sharing", that also shares WiFi profiles across the users machines as long as the same AzureID/LiveID is used.

Thanks a lot!

**Comment by si458** [comment]
@AFjolle https://github.com/Ylianst/MeshCentral/issues/new
i have just been given an old lenovo laptop running windows 10 which has AMT 8 on
and i do see the localProfileSynchronizationEnabled listed in the web ui on the AMT (meshcentral/meshcommander)
so i will have to have a play!
